src/EnergyPlus/StandardRatings.cc

```diff
@@ -6266,6 +6266,9 @@ namespace StandardRatings {
         Real64 e_sum(0.0);
         Real64 rh_sum(0.0);
 
+        // The minimum temperature below which the compressor is turned off
+        OATempCompressorOff = MinOATCompressor;
```

Comment:
This local variable was not set a value and the initialized value of `0` was used.